roxie/ccd/ccdserver.cpp

```diff
@@ -4559,18 +4563,24 @@ class CRemoteResultAdaptor : implements IEngineRowStream, implements IFinalRoxie
     {
         mu.clear();
         unsigned ctxTraceLevel = activity.queryLogCtx().queryTraceLevel();
+        unsigned lastActivity = msTick();
         for (;;)
         {
             checkDelayed();
             unsigned timeout = remoteId.isSLAPriority() ? slaTimeout : (remoteId.isHighPriority() ? highTimeout : lowTimeout);
```

Comment:
can this code, and the next new lines move outside the for(;;)?

Reply:
Probably. Repushed